DateInput/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import styles from './style.css'; // 'react-day-picker' uses global BEM-like classes
+import DayPicker from 'react-day-picker';
+
+const inThePast = (day) => {
+  const now = new Date();
+  const isToday = day.toDateString() === now.toDateString();
+  return (day < now) && !isToday;
+};
+
+const DateInput = ({
+  onChange,
+  value,
+}) =>
```

Comment:
@hharnisc - updated the terminology  here to 'value' to fit what redux-form will be expecting. I _think_ that was the only thing that we needed for this. Curious if this feels ready to go? Thanks! 😄